config.yaml

```diff
@@ -37,14 +41,14 @@ plugins:
         manage: 13370
         pub: 13371
         sub: 13372
-    cif3:
-      api:
-        host: http://localhost:5000
-        ssl: false
-        token: CIF_TOKEN
-      group: everyone
-      confidence: 7.5
-      tlp: amber
-      tags:
-        - test
-        - malicious
+    #cif3:
+    #  api:
+    #    host: http://localhost:5000
+    #    ssl: false
+    #    token: CIF_TOKEN
+    #  group: everyone
+    #  confidence: 7.5
+    #  tlp: amber
+    #  tags:
+    #    - test
+    #    - malicious
```

Comment:
Maybe move to a `config.yaml.example` instead of commenting?

Reply:
I created a `config.yaml.example` file and `gitignored` the `config.yaml`. The README is updated with a short sentence about the example file.